### PR TITLE
Feat_T31035: adjust design: header - diplanbau

### PR DESCRIPTION
**Pull request by sakutademos**
**Ticket:** https://yaits.demos-deutschland.de/T31035

**Description:** 

- this PR allows to configure external links section
- this PR adds `<span>` tag to the text in the `base_header_nav_item_simple_language.html.twig `and `base_header_nav_item_sign_language.html.twig` files which works as a wrapper and makes it possible to adjust required changes

_It is connected with the same PR in the DiPlanBau project._

- [x] Link all relevant tickets
- [x] Move the tickets on the board accordingly

### CODE REVIEW

File: demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml [hunk 1]
```diff
@@ -475,3 +475,11 @@ parameters:
     # This can be a relative or an absolute path, relative paths
     # will use DemosPlanPath::getRootDir() as base.
     deployment_sync_dir: '../htdocs'
+
+    # Allows to define links, that will be presented in a special section on the main page.
+    # Note that the data will be used in the frontend directly, i.e. no measures against potentially harmful
+    # scripts is taken if such thing is used here.
+    main_page_external_links:
+        DiPlan Portal: 'https://diplanportal-dev.demos-europe.eu/'
```
Comment (spiess-demos) [inline demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml:483]: Is the space intentional? Or should it be

```
        DiPlanPortal: 'https://diplanportal-dev.demos-europe.eu/'
```

Comment (sakutademos) [reply demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml:484]: It is with empty space in Figma --> 
image

Is it a bad practice to add empty space in the variables in `yaml` file?

Comment (spiess-demos) [reply demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml:484]: While it is allowed via the spec, as this [so answer implies](https://stackoverflow.com/questions/40977183/spaces-in-yaml-keys), i asked myself if it is a good idea to use the key as the link text. But we can also change this later, when it poses an issue.

File: demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml [hunk 2]
```diff
@@ -480,4 +480,6 @@ parameters:
     # external_links:
     #     Name of some website: https://link-to-the-website.net
     #     Some other website: https://link-to-some-other-website.net
-    external_links: []
+    external_links:
```
Comment (spiess-demos) [inline demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml:483]: The actual values should be defined in the project override, as they are project specific. Here, i'd use the empty array as a default.

**Review by spiess-demos — CHANGES_REQUESTED** [review]

File: demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml (hunk 2 above)
Comment (sakutademos) [reply demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml]: ok, done here https://github.com/demos-europe/demosplan-core/pull/852/commits/0caa555be3b7f1ab7fefd8d9661048d9ec65cdd9

**Review by spiess-demos — APPROVED** [review]
Lgtm!

File: demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml [hunk 3]
```diff
@@ -480,6 +480,4 @@ parameters:
     # external_links:
     #     Name of some website: https://link-to-the-website.net
     #     Some other website: https://link-to-some-other-website.net
-    external_links:
-        DiPlan Portal: 'https://diplanportal-dev.demos-europe.eu/'
-        DiPlanCockpit: 'http://hh.cockpit.diplanung.de'
+    external_links: []
```
[the remark this answers is not in the document]
Comment (sakutademos) [reply demosplan/DemosPlanCoreBundle/Resources/config/parameters_default.yml:483]: done here https://github.com/demos-europe/demosplan-project-diplanbau/pull/57/commits/ac6473542482a870a4747bbd41962bb45b3b6bd4

**Pull request merged**